data/json/items/tools.json

```diff
@@ -3223,6 +3223,7 @@
     "description": "This is a glass vial, split into two chambers.  The divider is removable, which will cause the chemicals to mix.  If this mixture is exposed to air (perhaps by throwing the vial) they will spill out as a pool of potent acid.",
     "price": 5000,
     "material": "glass",
+    "flags": "PROCESS_ON_DROP",
```

Comment:
Please don't forget to add the description of the introduced flag(s) to the documentation.